### PR TITLE
add active wallets metrics

### PR DESCRIPTION
**Pull request by 0xissui**
# Description

Add active wallet metrics table

# Tests 

- [x] Please provide evidence of your successful `dbt run` / `dbt test` here
image

- [ ] Any comparison between `prod` and `dev` for any schema change

# Checklist
- [ ] Follow [dbt style guide](https://github.com/dbt-labs/corp/blob/main/dbt_style_guide.md)
- [ ] Tag the person(s) responsible for reviewing proposed changes @chuxinh 
- [ ] Notes to deployment, if a `full-refresh` is needed for any table

### CODE REVIEW

**Comment by 0xissui** [comment]
ah, lmk if we need to have it as table instead of incremental

File: models/metrics/metrics_active_wallets.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{{
+    config(
+        materialized='incremental',
```
Comment (chuxinh) [inline models/metrics/metrics_active_wallets.sql:3]: yea would prefer to have it as a table instead! Thanks!!

File: models/metrics/metrics_active_wallets.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}}
+
+with active_wallets as (
+
+    select
+
+        date_trunc('day', block_timestamp) as date,
+        count(distinct tx_signer) as daily_active_wallets,
+        sum(daily_active_wallets) over (order by date ROWS BETWEEN 6 PRECEDING AND CURRENT ROW) as weekly_active_wallets,
+        sum(daily_active_wallets) over (order by date ROWS BETWEEN 29 PRECEDING AND CURRENT ROW) as monthly_active_wallets
```
Comment (chuxinh) [inline models/metrics/metrics_active_wallets.sql:18]: lower case to be consistent with out style guide

File: models/metrics/metrics_active_wallets.sql [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    )
+}}
+
+with active_wallets as (
+
+    select
+
+        date_trunc('day', block_timestamp) as date,
+        count(distinct tx_signer) as daily_active_wallets,
+        sum(daily_active_wallets) over (order by date ROWS BETWEEN 6 PRECEDING AND CURRENT ROW) as weekly_active_wallets,
```
Comment (chuxinh) [inline models/metrics/metrics_active_wallets.sql:17]: maybe also name it as `rolling_7day_active_wallets` and `rolling_30day_active_wallets` instead

File: models/metrics/metrics_active_wallets.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+
+version: 2
+
+models:
+  - name: metrics_active_wallets
+    description: "Daily, weekly, and monthly active wallets on NEAR."
```
Comment (chuxinh) [inline models/metrics/metrics_active_wallets.yml:6]: update here as well in terms of rolling 7 day and 30 day

**Review by chuxinh — CHANGES_REQUESTED** [review]

**Review by chuxinh — APPROVED** [review]
LGTM!

**Pull request merged**